### ISSUE TITLE
Some questions on the paper

### ISSUE THREAD

**Issue by jessyford** [MLE-Dojo/MLE-Dojo#1]
Hi, thanks for your great work, however I have some confusions after reading your paper.

1. In Fig1 there are 22 categories in the figure, but in section 3.1 you mentioned 'yielding a total of 15 distinct task types' ?

2. In the paper you mentioned several times that MLE-Dojo have more than 200 competitions, but in the last paragraph of section 3.2 you mentioned 'We partition MLE-Dojo into training and evaluation subsets with a 150 : 50 split.', so are there exactly 200 competitions in MLE-Dojo or more than 200? or they are not same thing?

Furthermore, in X-Gym section of Related Work, you mentioned 'MLE-Dojo significantly extends both the quantity and complexity of tasks, offering 200+ competitions with an additional dedicated training set of 150 tasks.'
This makes me even more confused. So what is the actual picture of the entire data structure of MLE-Dojo? Is it a 200+ competitions plus 150 additional tasks, so basically 350+ tasks in total? If so, then where is the 50 tasks in '150:50' split?

3. In table 1 you checked 'Training Facilities' for MLE-Dojo, and in the paper you mentioned splitting training and testing set. So does MLE-Dojo project actually support complete pipeline of SFT and RL? However I didn't see any further information in the paper or in the repo, such as training result of any model on the training set with MLE-Dojo project.

4. I also have some confusion about how you split training set and testing set, for example I am not sure if you split mle-bench dataset into train set and test set at the same time, etc. But I assume this might be related to my Question 2.

Thanks again for your help!

**Comment by rushi-Q** [comment]
Thank you for your questions.

Regarding Question 1: There is no universally defined taxonomy for categorizing tasks into specific categories—the figure is mainly illustrative. You can refer to Appendix D for all the detailed information about the data.

Regarding Questions 2 and 4: As for the total number of tasks, we have currently released 200 tasks, split into a 150:50 ratio. The complete task list and the specific split can be found in Appendix D. There are more to release in the future. We also welcome community participation and contributions.

Regarding Question 3: MLE-dojo provides the Gym-like ground for SFT and RL training. In the current release, our code supports sampling training trajectory data, making it easy to integrate with various open-source SFT/RL training frameworks. You can refer to sampled trajectories in  https://github.com/MLE-Dojo/MLE-Dojo/tree/main/trajectories. We consider to integrate and release mle-dojo training code into this repository.

If you have any other questions, feel free to email rqiang6@gatech.edu for further discussion.
